src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -541,6 +567,10 @@ def construct_containerapp(self):
             else:
                 set_managed_identity(self.cmd, self.get_argument_resource_group_name(), self.containerapp_def, user_assigned=[self.get_argument_registry_identity()])
 
+        if self.get_argument_source():
+            app = self.set_up_create_containerapp_if_source_or_repo()
+            self.containerapp_def = self.set_up_create_containerapp_source(app=app)
```

Comment:
As set_up_create_containerapp_source no need and has not return containerapp, here need to remove.

``` suggestion
self.set_up_create_containerapp_source(app=app)
```

Reply:
Updated